vsi/src/scan.rs

```diff
@@ -68,6 +71,17 @@ impl Display for Artifact {
 }
 
 impl Artifact {
+    /// Normalize the artifact's path to use unix-style path separators.
+    pub fn normalize(self) -> Self {
```

Comment:
[Optional] [prefer] unit test for this function

Reply:
[Q] Do we need to worry about [windows UNC/DOS paths conversion](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats#example-ways-to-refer-to-the-same-file)? I presume this is not an issue for us.

Reply:
> Do we need to worry about [windows UNC/DOS paths conversion](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats#example-ways-to-refer-to-the-same-file)?

Not in this case, as the paths being normalized are relative to the scan directory.